### ISSUE TITLE
Separating model output in combined model fits (Trac #209)

### ISSUE THREAD

**Issue by butlerpd** [SasView/sasview#357]
When fitting to a combined S(Q) P(Q) for example the plot of the model can be saved as points to file but the individual components (i.e. S(Q) or P(Q) cannot.

Migrated from http://trac.sasview.org/ticket/209
```json
{
    "status": "closed",
    "changetime": "2017-01-05T11:13:57",
    "_ts": "2017-01-05 11:13:57.637462+00:00",
    "description": "When fitting to a combined S(Q) P(Q) for example the plot of the model can be saved as points to file but the individual components (i.e. S(Q) or P(Q) cannot.",
    "reporter": "butler",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView Fitting Redesign",
    "time": "2014-03-11T14:30:01",
    "component": "SasView",
    "summary": "Separating model output in combined model fits",
    "priority": "critical",
    "keywords": "",
    "milestone": "SasView 4.1.0",
    "owner": "mathieu",
    "type": "enhancement"
}
```

**Comment by butlerpd** [comment]
Trac update at `2014/04/01 11:05:01`: **butler** changed milestone from "SasView 3.0.0" to "SasView Next Release +1"

**Comment by butlerpd** [comment]
Trac update at `2015/01/30 01:00:34`: **butler** changed workpackage from "" to "SasView Fitting Redesign"

**Comment by RichardHeenan** [comment]
Trac update at `2016/03/24 18:32:16`: **richardh** commented:

Had another request from a user (Sarah Rogers) for S(Q) output, she went off to use FISH instead ...

How hard would if be to generate a plot of P(Q) and S(Q) in the same manner that the residuals plot is produced?

Richard

**Comment by butlerpd** [comment]
Trac update at `2016/06/10 17:52:54`: **butler** commented:

Had another request from a user at Monash University (Vikram Raghuwanshi) for S(q), P(Q) output but also the polydispersity distribution function.  This should probably be escalated to a high priority for work at the next code camp at ORNL.

**Comment by butlerpd** [comment]
Trac update at `2016/06/10 17:53:12`: **butler** changed priority from "major" to "critical"

**Comment by butlerpd** [comment]
Trac update at `2016/10/04 02:13:12`: **butler** changed milestone from "SasView Next Release +1" to "SasView 4.1.0"

**Comment by mdoucet** [comment]
Trac update at `2016/10/07 13:15:07`: 
* **mathieu** changed resolution from "" to ""
* **mathieu** changed status from "new" to "reopened"

**Comment by mdoucet** [comment]
Trac update at `2016/10/07 13:15:31`: 
* **mathieu** changed resolution from "" to "fixed"
* **mathieu** changed status from "reopened" to "closed"

**Comment by mdoucet** [comment]
Trac update at `2016/10/07 13:16:14`: 
* **mathieu** changed resolution from "fixed" to ""
* **mathieu** changed status from "closed" to "reopened"

**Comment by mdoucet** [comment]
Trac update at `2016/10/07 13:16:41`: 
* **mathieu** changed owner from "Mathieu Doucet <doucetm@ornl.gov" to "mathieu"
* **mathieu** changed status from "reopened" to "assigned"

**Comment by mdoucet** [comment]
Trac update at `2016/10/07 13:17:39`: 
* **mathieu** changed resolution from "" to "fixed"
* **mathieu** changed status from "assigned" to "closed"

**Comment by RichardHeenan** [comment]
Trac update at `2016/11/15 16:14:32`: 
* **richardh** commented:

At least on the builds I am making, the separate storage of P(Q) and S(Q) in the "data explorer" panel has disappeared, no doubt due to changes made to the product model?
Richard



* **richardh** changed resolution from "fixed" to ""
* **richardh** changed status from "closed" to "reopened"

**Comment by smk78** [comment]
Trac update at `2016/11/16 21:13:29`: **smk78** commented:

I concur with richardh.

**Comment by mdoucet** [comment]
Trac update at `2016/12/29 18:58:02`: 
* **mathieu** commented:

In changeset b0bebdcbbd448e8576b27b010070a35fda67932a:

```text
#!CommitTicketReference repository="sasview" revision="b0bebdcbbd448e8576b27b010070a35fda67932a"
Adapt SasViewModel to new multiplication model replacement. Fixes #357
```


* **mathieu** changed resolution from "" to "fixed"
* **mathieu** changed status from "reopened" to "closed"

**Comment by smk78** [comment]
Trac update at `2017/01/05 11:13:57`: **smk78** commented:

Tested fix on W7 local build. Passes. P(Q) & S(Q) appear as additional theories in Data Explorer (if you expand the selected data) and can be plotted. P(Q) & (SQ) can also be written to file from the Graph context menu or the Data Explorer context menu.
